### PR TITLE
Add CCTP Bridger

### PR DESCRIPTION
**Pull request by cwang25**

### CODE REVIEW

File: src/DeFiScripts.sol [hunk 1]
```diff
@@ -326,3 +327,16 @@ contract ApproveAndSwap {
         IERC20(sellToken).forceApprove(to, 0);
     }
 }
+
+contract CCTPBridgeActions {
```
Comment (kevincheng96) [inline src/DeFiScripts.sol:331]: I think this should be in its own file since it's not really related to DeFi.

**Comment by cwang25** [comment]
`[FAIL. Reason: setup failed: Could not instantiate forked environment with fork url: https://node-provider.compound.finance/ethereum-mainnet/] setUp() (gas: 0)`

Looks like this repo hasn't set up the proper forking access to Infura?

File: test/CCTPBridge.t.sol
Comment (kevincheng96) [inline test/CCTPBridge.t.sol:1]: File should be named `CCTPBridgeActions.t.sol`.

File: test/CCTPBridge.t.sol [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                CCTPBridgeActions.bridgeUSDC,
+                (tokenMessenger, 500_000e6, 6, bytes32(uint256(uint160(address(wallet)))), USDC)
+            ),
+            ScriptType.ScriptSource
+        );
+        (uint8 v, bytes32 r, bytes32 s) = new SignatureHelper().signOp(alicePrivateKey, wallet, op);
+        assertEq(IERC20(USDC).balanceOf(address(wallet)), 1_000_000e6);
+        vm.resumeGasMetering();
+        wallet.executeQuarkOperation(op, v, r, s);
+        assertEq(IERC20(USDC).balanceOf(address(wallet)), 500_000e6);
```
Comment (kevincheng96) [inline test/CCTPBridge.t.sol:68]: nit: add an extra line between the action and assertions

**Review by kevincheng96 — COMMENTED** [review]
LGTM! Left some comments

File: src/DeFiScripts.sol (hunk 1 above)
Comment (cwang25) [reply src/DeFiScripts.sol:331]: That's true..
Will make it into separate file then 👍

**Comment by kevincheng96** [comment]
> Looks like this repo hasn't set up the proper forking access to Infura?

I just added the `NODE_PROVIDER_BYPASS_KEY` secret to this repo. Let's give it another run and see if it works.

File: test/CCTPBridge.t.sol
Comment (cwang25) [reply test/CCTPBridge.t.sol:1]: Ok 👍

File: test/CCTPBridge.t.sol (hunk 2 above)
Comment (cwang25) [reply test/CCTPBridge.t.sol]: ok 👍

**Comment by cwang25** [comment]
This repo is not `format`,  
so for the lint error I think will need a separate PR for "fmt"  might be good, cuz those fmt errors are not in the scope of this PR.

File: src/DeFiScripts.sol [hunk 3]
```diff
@@ -9,6 +9,7 @@ import {QuarkScript} from "quark-core/src/QuarkScript.sol";
 
 import {IComet} from "./interfaces/IComet.sol";
 import {ICometRewards} from "./interfaces/ICometRewards.sol";
+import {ITokenMessenger} from "./interfaces/ITokenMessenger.sol";
```
Comment (kevincheng96) [inline src/DeFiScripts.sol:12]: This can be removed

**Review by kevincheng96 — APPROVED** [review]

**Pull request merged**